src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/CSharpFormatter.cs

```diff
@@ -48,26 +52,26 @@ internal class CSharpFormatter
             RazorCodeDocument codeDocument,
             Range range,
             Uri uri,
-            FormattingOptions options)
+            FormattingOptions options,
+            bool formatOnClient = false)
         {
-            if (!_documentMappingService.TryMapToProjectedDocumentRange(codeDocument, range, out var projectedRange))
+            Range projectedRange = null;
```

Comment:
Why pull this out of the `out var` format?

Reply:
C# doesn't seem to be smart in this case. It complains about projectedRange being unassigned below.